commitlint.config.js

```diff
@@ -1,22 +1,23 @@
-const {
-  default: getWorkspacePackages,
-} = require('@pnpm/find-workspace-packages')
+'use strict'
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
 
-async function getPackages(context) {
-  const ctx = context || {}
-  const cwd = ctx.cwd || process.cwd()
-  const packages = await getWorkspacePackages(cwd)
-  return packages
-    .map((pkg) => pkg.manifest.name)
-    .filter((name) => !!name)
-    .map((name) => (name.charAt(0) === '@' ? name.split('/')[1] : name))
-}
+const getPackages = (packagePath) =>
+  fs
```

Comment:
IMHO, we can use `fast-glob` here to simplify code.
```ts
fg.sync("*", { cwd: packagePath, onlyDirectories: true })
```

Reply:
> IMHO, we can use `fast-glob` here to simplify code.
> 
> ```ts
> fg.sync("*", { cwd: packagePath, onlyDirectories: true })
> ```

Great !!! have done